Analysis/tfcrig/files.py

```diff
@@ -92,10 +92,6 @@ def clean(self) -> None:
         self._examine_and_fix_typos_in_data_files()
         self._sync_second_mouse()
 
-    # Testing purposes - to test individually
```

Comment:
This can be removed because it is commented out, and because it can always be accessed with the following:

```python
files = RigFiles(
    data_root=DATA_ROOT,
    dry_run=False,
)
files._sync_second_mouse()
```